schedule/yam/agama_gnome_unattended_ppc64le.yaml

```diff
@@ -7,6 +7,5 @@ schedule:
   - yam/agama/patch_agama_tests
   - yam/agama/agama
   - installation/grub_test
-  - installation/first_boot
```

Comment:
Add comments for this file also. for the reason we remove first_boot on ppc64le.